trackma/tracker/inotifyBase.py

```diff
@@ -42,10 +42,12 @@ def _proc_poll(self):
             if not p.isdigit(): continue
 
             # Get process name
-            with open('/proc/%s/cmdline' % p, 'rb') as f:
-                cmdline = f.read()
-                pname = cmdline.partition(b'\x00')[0]
-
+            try:
+                with open('/proc/%s/cmdline' % p, 'rb') as f:
+                    cmdline = f.read()
+                    pname = cmdline.partition(b'\x00')[0]
+            except FileNotFoundError:
+                continue            # Continue without raising exception if process doesn't exist.
```

Comment:
This comment is entirely redundant. You can see everything it describes by just reading the code.

Reply:
I've removed the comment.